gateway/tests/fixtures/files_fixtures.json

```diff
@@ -79,6 +79,20 @@
             ]
         }
     },
+    {
```

Comment:
@korgan00 I think we are not using `ProgramDelete` in our tests so you can delete it